pkg/awsutil/config.go

```diff
@@ -43,11 +47,15 @@ func Config() (*aws.Config, error) {
 		}
 	}
 
+	if os.Getenv(envAwsSessionToken) != "" {
+		awsSessionToken = os.Getenv(envAwsSessionToken)
+	}
```

Comment:
This conditional is redundant. Furthermore, I think we can pass `os.Getenv(envAwsSessionToken)` directly to `credentials.NewStaticCredentials` instead of defining a variable `awsSessionToken`. I don't see a reason we need that global variable.